<ui version="4.0" >
 <class>ProjectSettings_UI</class>
 <widget class="QDialog" name="ProjectSettings_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Project Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_11" >
     <property name="text" >
      <string>Project folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="KUrlRequester" name="kurlrequester" />
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QGroupBox" name="properties" >
     <property name="title" >
      <string>Video Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KComboBox" name="profiles_list" />
      </item>
      <item row="1" column="0" colspan="2" >
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="p_size" >
          <property name="text" >
           <string>720x576</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Frame rate:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLabel" name="p_fps" >
          <property name="text" >
           <string>25/1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Aspect ratio:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="p_aspect" >
          <property name="text" >
           <string>59/54</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>Display ratio:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QLabel" name="p_display" >
          <property name="text" >
           <string>4/3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="p_progressive" >
        <property name="text" >
         <string>Interlaced</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Thumbnails:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QCheckBox" name="video_thumbs" >
     <property name="text" >
      <string>Video</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QCheckBox" name="audio_thumbs" >
     <property name="text" >
      <string>Audio</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>229</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="4" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>105</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectSettings_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectSettings_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
